pkg/controllers/disruption/consolidation_test.go

```diff
@@ -79,6 +80,20 @@ var _ = Describe("Consolidation", func() {
 				Allocatable: map[v1.ResourceName]resource.Quantity{v1.ResourceCPU: resource.MustParse("32")},
 			},
 		})
+		spotNodeClaim, spotNode = test.NodeClaimAndNode(v1beta1.NodeClaim{
```

Comment:
The diff shown by github is huge unfortunately for this file. But the actual changes are minimal. Each test has been refactored to avoid duplicating the logic each for on-demand and spot instances consolidation testing.